demos/demo_common/polyfills.js

```diff
@@ -0,0 +1,48 @@
+// Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#Polyfill
```

Comment:
Not sure if this has to run on-prem w/o internet access, but one solution we have used is to leverage https://polyfill.io/ - whats nice is that the service will only return the needed polyfills for the browser... so if it's chrome that makes the request, it returns an empty script, vs IE9, where it returns a mess of JS to polyfill all the things. You can also specify the things you want - again to minimize what's packaged and sent. Just an option...

Reply:
We do have to run w/o internet access; this is one of the reasons that switching to Hub's templates over references to AGOL source items was hugely useful.

Reply:
Cool - for our Portal builds we package in the polyfills just like this :)